invenio_app_rdm/administration/records/records.py

```diff
@@ -100,10 +101,11 @@ class DraftAdminListView(AdminResourceListView):
     """Configuration for OAI-PMH sets list view."""
 
     api_endpoint = "/user/records"
-    name = "Drafts"
+    name = "drafts"
     resource_config = "records_resource"
     search_request_headers = {"Accept": "application/vnd.inveniordm.v1+json"}
     title = "Drafts"
+    menu_label = "Drafts"
```

Comment:
```suggestion
    menu_label = _("Drafts")
```

Reply:
i thought that too, but i would keep the translation fix combined in the #2886

Reply:
If we can make it in one go, why split it into two changes for something so small? :)